lib/Infer/ExhaustiveSynthesis.cpp

```diff
@@ -66,6 +66,9 @@ namespace {
   static cl::opt<bool> EnableDataflowPruning("souper-dataflow-pruning",
     cl::desc("Enable pruning based on dataflow analysis (default=false)"),
     cl::init(false));
+  static cl::opt<bool> SynthesisConstWithCegisLoop("souper-synthesis-const-with-cegis",
+    cl::desc("Synthesis constants with CEGIS (default=false)"),
```

Comment:
I think default this to true since the exists-forall method isn't working well

Reply:
This method is also not working well for now, since no input specialization is implemented.

Say it cannot synthesis mul(x, 100) with less than 100 guess, when the solver guesses from 0 and increase the guess by 1 each time. Input 0 will always make the check satisfied for any constant.